about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Global school about page</title>
+ <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
+ <link rel="stylesheet" href="css/mobile_style.css">
+ <link rel="stylesheet" href="css/Destop_style.css">
+ <link rel="stylesheet" href="css/about_style.css">
+</head>
+<body>
+ <header>
+  <div class="logo">
+   <a href="index.html"><span>GLO</span>BAL SCH<span>OOL</span></a>
+  </div>
+  <nav>
+   <ul>
+    <li><a href="index.html">Home</a></li>
+    <li><a href="about.html">About</a></li>
+    <li><a href="#">Join</a></li>
+    <li><a href="#">Sponsor</a></li>
+    <li><a href="#">News</a></li>
+   </ul>
+   <div class="menu_bar">
+    <i class="fas fa-bars"></i>
+   </div>
+  </nav>
+ </header>
+ <main>
+  <section class="first_sec">
+   <div>
+    <p class="wel">"Welcome to GLOBAL SCHOOL"</p>
+    <h1 class="title">LET CONTRIBUTE TO THE DEVELOPPEMENT OF OUR COMMUNITY</h1>
+    <p class="intro">The Global School brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two.
+     This year, we hope to expand our invitation list including organizations and individuals.
+    </p>
+    <p class="hrs">Please contact us for any further questions about GLOBAL SCHOOL</p>
```

Comment:
- [ ] You have this line 39 sentence is `bold`, while in the given design it is not bold. I recommend you follow the design guidelines.